src/main/java/ru/geekbrains/java/evgeny/lessonone/LessonOne.java

```diff
@@ -2,6 +2,91 @@
 
 public class LessonOne {
     public static void main(String[] args) {
-        System.out.println("Hello World!!");
+        System.out.println("Hello world!!");
+        //Создаем переменные всех типов с их инциализацией
+        int a = 20;
+        byte b = 120;
+        short c = 12434;
+        long d = 214354;
+        double e = 1923.00;
+        float f = 123.9797f;
+        char g = 'C';
+        boolean h = true;
+
+        System.out.println();
+        System.out.println(result(12.7f, 10.9f,15.6f,17.2f));
+        System.out.println();
+        System.out.println(result10and20(5,4));
+        System.out.println();
+        negativeOrNot(5);
+        System.out.println();
+        System.out.println(isNegative(-6));
+        helloName("Evgeny");
+        checkYear(2028);
+
+
+
+
+
     }
+
+    /*
+    Написать метод, который определяет, является ли год високосным, и выводит сообщение в консоль.
+    Каждый 4-й год является високосным, кроме каждого 100-го, при этом каждый 400-й – високосный.
+     */
+
+    private static void checkYear(int year) {
+        if ((year % 4 == 0) && (0 == year % 400) || !(year % 100 == 0) ){
+            System.out.println(year + " год високосный");
+        } else System.out.println(year + " год не високоссный");
+
+    }
+
+    /*
+        Написать метод, которому в качестве параметра передается строка,
+        обозначающая имя. Метод должен вывести в консоль сообщение «Привет, указанное_имя!».
+     */
+    private static void helloName(String name) {
+        System.out.println("Привет " + name);
+    }
+
+    /*
+    Написать метод, которому в качестве параметра передается целое число.
+    Метод должен вернуть true, если число отрицательное, и вернуть false если положительное.
+     */
+    private static boolean isNegative(int a) {
+        if (a < 0){
+            return true;
+        }else return false;
+    }
+
+    /*
+    Написать метод, принимающий на вход два целых числа и проверяющий,
+    что их сумма лежит в пределах от 10 до 20 (включительно),
+    если да – вернуть true, в противном случае – false
+     */
+    private static void negativeOrNot(int a) {
+        if (a >=0 ){
+            System.out.println(a + " положительное");
+        } else {
+            System.out.println(a + " отрицательное");
+        }
+    }
+
+    /*
+    Написать метод, принимающий на вход два целых числа и проверяющий,
+     что их сумма лежит в пределах от 10 до 20 (включительно),
+     если да – вернуть true, в противном случае – false
+     */
+    private static boolean result10and20(int a, int b) {
+        if((a+b)>10 && (a+b)<=20){
```

Comment:
```suggestion
        return (a+b)>10 && (a+b)<=20;
```
логическое выражение само по себе является логическим значением, его можно использовать в качестве результата